### DISCUSSION TITLE
Фото/Видео в постах/Альбом в группе

### DISCUSSION THREAD

**Discussion by KaDim2515** [OpenVK/openvk#485] [Ideas] [answered]
Доброго времени суток!
Хотелось бы в ближайшем времени узреть возможность добавлять к посту более чем 1 фото. 
У меня группа на тематику фото/видео и мне крайне неудобно делать подборки. 
Я думаю исправление этой ситуации улучшило бы восприятие  информации в посте пользователем  OPENVK. 
Так же в группе почему-то отсутствует возможность создания альбома.

**Comment by veselcraft** [comment, marked as answer]
[раз](https://github.com/openvk/openvk/issues/427) и [два](https://github.com/openvk/openvk/tree/textarea-refactor)
